fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -33,6 +33,14 @@ class TransporterExecutor
     def execute(command, hide_output)
       return command if Helper.test?
 
+      # Workaround because the traditional transporter broke on 1st March 2018
+      # More information https://github.com/fastlane/fastlane/issues/11958
+      # As there was no communication from Apple,we don't know if this is a temporary
```

Comment:
space between `Apple,we`

Reply:
My space key stopped working, it's annoying